src/test/scala/centaur/CallCacheSpec.scala

```diff
@@ -26,6 +28,18 @@ class CallCacheSpec extends FlatSpec with Matchers with ParallelTestExecution {
     }
   }
 
+  "dockerHashNoLookup" should "not lookup docker tag if backend doesn't support docker" in {
+    Workflow.fromPath(DockerHashNoLookup) match {
+      case Valid(w) =>
+        if (w.backends forall CromwellBackendsCompanion.supportedBackends.contains){
+          TestFormulas.runSequentialCachingWorkflows(w, w).run.get
+        } else {
+          succeed
```

Comment:
yeah I don't know how to get it to run if it's found and ignored otherwise. If you or maybe @geoffjentry knows how to do that it would be way better.